Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -437,6 +445,114 @@ badgeCardView config push =
     ] <> FontStyle.body4 TypoGraphy
   ]
 
+driverUpiQrCodeView :: forall w. Config -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w 
+driverUpiQrCodeView config push = 
+  linearLayout 
+    [
+      height WRAP_CONTENT
+    , width MATCH_PARENT
+    , background Color.white900
+    , orientation VERTICAL
+    , cornerRadius 16.0
+    , gravity CENTER
+    , padding $ Padding 16 16 16 16
+    , margin $ MarginBottom 26
+    ][
+      textView $
+      [
+        text config.driverUpiQrCard.text
+      ] <> FontStyle.subHeading1 TypoGraphy
+      , imageView [
+          height $ V 165
+        , width $ V 165
+        , margin $ MarginVertical 8 13
+        , id $ getNewIDWithTag config.driverUpiQrCard.id
+        , afterRender push (const (UpiQrRendered $ getNewIDWithTag config.driverUpiQrCard.id))
```

Comment:
don't rely on afterRender try switching it to onAnimationEnd

Reply:
onAnimationEnd is an hack he it doesn't make sense to start animation and use onAnimationEnd